src/main/java/org/folio/consortia/service/impl/UserServiceImpl.java

```diff
@@ -99,6 +100,7 @@ public User prepareShadowUser(UUID userId, String tenantId) {
           personal.setPreferredContactTypeId(userPersonal.getPreferredContactTypeId());
           user.setPersonal(personal);
         }
+        user.setType(SHADOW_USER_TYPE);
```

Comment:
Lets wait a bit and confirm do we need to fill userPersonal at all, because now when we agreed to not show shadow users in UI at all - this does not make sense to fill userPersonal, and the less personal data we expose to other tenants - the better.

Reply:
@SinghAdes regarding userPersonal - please delete it, we will pull all required user data from original tenant if have some view for shadow users in future.
Also we need in someway to set originalTenantId where user was created, thinking where we can set it, maybe in additionalProperties

Reply:
last name can be required field, need to check, if so - we need to remain it